hack/addons/kustomize/cluster-autoscaler/helm-values.yaml

```
@@ -0,0 +1,39 @@
+# Copyright 2023 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+---
+# This is a hack, but because a single cluster-autoscaler deployment can only monitor a single Kubernetes cluster
+# we expect 'tmpl-clustername-tmpl' and 'tmpl-clusternamespace-tmpl'
+# to be replaced with the Cluster's name and namespace.
+fullnameOverride: cluster-autoscaler-tmpl-clustername-tmpl
```

What about using go template syntax? And parsing the CRS configmap through go gemplating when creating the actual resources?
```suggestion
fullnameOverride: cluster-autoscaler-{{ .clusterName }}
```

---

Oh let's stick with what you've done here - it's safe enough.